src/query/service/src/pipelines/processors/transforms/transform_udf_script.rs

```diff
@@ -47,93 +51,261 @@ impl TransformUdfScript {
+                code,
+                &func.func_name,
+            )
+            .map_err(|err| ErrorCode::UDFDataError(format!("Cannot add js function: {err}")))?;
+
+        let _ = js_runtime
+            .call(&func.name, input_batch)
+            .map(|result_batch| self.update_datablock(func, result_batch, data_block))
+            .map_err(|err| {
+                ErrorCode::from_string(format!(
+                    "JS Runtime Call {} execution Error {err}",
+                    func.name
+                ))
+            })?;
+
+        Ok(())
+    }
+
+    // TODO: Enable this section after successfully building `arrow_udf_python`.
+    fn handle_python(
+        &self,
+        func: &UdfFunctionDesc,
+        input_batch: &RecordBatch,
+        data_block: &mut DataBlock,
+        code: &str,
+    ) -> Result<()> {
+        log::info!("Executing Python UDF: {}", func.name);
+
+        let tmp_schema = DataSchema::new(vec![DataField::new(
+            "result",
+            func.data_type.as_ref().clone(),
+        )]);
+        let arrow_schema = Schema::from(&tmp_schema);
+
+        let mut py_runtime = arrow_udf_python::Runtime::new().map_err(|err| {
+            ErrorCode::UDFDataError(format!("Cannot create python runtime: {err}"))
+        })?;
+
+        py_runtime
+            .add_function_with_handler(
+                &func.name,
+                arrow_schema.field(0).data_type().clone(),
+                arrow_udf_python::CallMode::ReturnNullOnNullInput,
+                code,
+                &func.func_name,
+            )
+            .map_err(|err| ErrorCode::UDFDataError(format!("Cannot add py function: {err}")))?;
+
+        let _ = py_runtime
+            .call(&func.name, input_batch)
+            .map(|result_batch| self.update_datablock(func, result_batch, data_block))
+            .map_err(|err| {
+                ErrorCode::from_string(format!(
+                    "Py Runtime Call {} execution Error {err}",
+                    func.name
+                ))
+            })?;
+
+        Ok(())
+    }
+
+    fn handle_wasm(
+        &self,
+        func: &UdfFunctionDesc,
+        input_batch: &RecordBatch,
+        data_block: &mut DataBlock,
+        code: &str,
+    ) -> Result<()> {
+        let code = code.trim();
+
+        log::info!("Executing WASM Module: {} UDF: {}", code, func.name);
+
+        let rt = tokio::runtime::Builder::new_multi_thread()
+            .worker_threads(1)
+            .enable_all()
+            .build()
+            .unwrap();
+
+        // Execute the async block using the created runtime
+        let compressed = rt.block_on(async {
+            let operator: Operator = DataOperator::instance().operator();
+            operator.read(code).await.map_err(|err| {
+                ErrorCode::from_string(format!("Failed to read Wasm Module {} {}", code, err))
+            })
+        })?;
+
+        let mut decoder = DecompressDecoder::new(CompressAlgorithm::Zstd);
```

Comment:
Maybe we don't need to consider compression in this pr I think.  Do you know the size of the file result in your test case?

Let's store the plain text code of wasm script build result if the size is less than 4MB.

Reply:
Code changes completed and ready for review.